doc/rtd/explanation/instancedata.rst

```diff
@@ -65,7 +65,6 @@ Using ``instance-data``
 
 ``instance-data`` can be used in:
 
-* :ref:`User data scripts<user_data_script>`.
```

Comment:
This was redundant, since user data scripts are documented on the user data formats page.